package.json

```diff
@@ -42,14 +42,14 @@
     "publish-canary": "git checkout main && git pull && lerna version prerelease --preid canary --message \"Publish Canary\" --exact",
     "publish-from-github": "./utils/publish.sh",
     "changelog": "node utils/changelog.js",
-    "build": "node utils/gen.js && turbo run build",
+    "build": "node utils/gen.js && NO_UPDATE_NOTIFIER=1 turbo run build",
     "vercel-build": "pnpm build && pnpm run pack && cd api && node -r ts-eager/register ./_lib/script/build.ts",
     "pre-commit": "lint-staged",
     "test": "jest --rootDir=\"test\" --testPathPattern=\"\\.test.js\"",
-    "test-unit": "pnpm test && node utils/gen.js && turbo run test-unit",
-    "test-integration-cli": "node utils/gen.js && turbo run test-integration-cli",
-    "test-integration-once": "node utils/gen.js && turbo run test-integration-once",
-    "test-integration-dev": "node utils/gen.js && turbo run test-integration-dev",
+    "test-unit": "pnpm test && node utils/gen.js && NO_UPDATE_NOTIFIER=1 turbo run test-unit",
+    "test-integration-cli": "node utils/gen.js && NO_UPDATE_NOTIFIER=1 turbo run test-integration-cli",
+    "test-integration-once": "node utils/gen.js && NO_UPDATE_NOTIFIER=1 turbo run test-integration-once",
+    "test-integration-dev": "node utils/gen.js && NO_UPDATE_NOTIFIER=1 turbo run test-integration-dev",
```

Comment:
This will fail on windows, won't it? Maybe we should use [cross-env](https://github.com/kentcdodds/cross-env#readme) or some other solution.